plugins/org.python.pydev.refactoring/src/org/python/pydev/refactoring/ast/visitors/position/IndentVisitor.java

```diff
@@ -90,7 +90,14 @@ public Object visitAssert(Assert node) throws Exception {
 
     @Override
     public Object visitAssign(Assign node) throws Exception {
-        visit(node.targets[0]);
+        if (node.targets.length > 0) {
```

Comment:
I think that the proper fix here would be:

```
if(node.targets != null){
    for(SimpleNode t:node.targets){
        visit(t);
    }
}
```

Reply:
That's right. I just kept the original code

Reply:
Actually

1. node.targets is not stored as null
2. `i` is being used to check if there's more than one assign

```
if (i >= 1) { //more than one assign
    doc.add(lowerAndHigher.o2.getLine(), lowerAndHigher.o2.getBeginCol(),
        this.prefs.getAssignPunctuation(), node);
}
```